gulpfile.js

```diff
@@ -410,3 +411,33 @@ gulp.task('prod', function(callback) {
 gulp.task('dev:runserver', function(callback) {
     runSequence('dev', 'connect', 'watch', 'test:watch', callback);
 });
+
+//Image Optimization
+var imagemin = require('gulp-imagemin');
+var cache = require('gulp-cache');
+ gulp.task('images', function() {
+  return gulp.src('frontend/src/images/**/*')
+    .pipe(cache(imagemin({ optimizationLevel: 5, progressive: true, interlaced: true })))
+    .pipe(gulp.dest('frontend/src/images'));
+});
+
+//Image Optimization of Organization Folder
+var imagemin = require('gulp-imagemin');
+var cache = require('gulp-cache');
+ gulp.task('imagesorgan', function() {
+  return gulp.src('frontend/src/images/organizations/**/*')
+    .pipe(cache(imagemin({ optimizationLevel: 5, progressive: true, interlaced: true })))
+    .pipe(gulp.dest('frontend/src/images/organizations'));
```

Comment:
Src should be Dist folder.

Reply:
A new Dist folder should be made?? Because I cannot see any folder for that.

Reply:
Have you done with the setup of EvalAI frontend according to read me? Also, have you started gulp server?

Reply:
Yeah! I have set up. and I have done according https://andy-carter.com/blog/a-beginners-guide-to-the-task-runner-gulp

Reply:
So, if you done it correctly then it should have created a dist folder in your frontend directory. 
Please check or what tasks you are running to start a server?

Reply:
I will try to setup from the beginning once more and will update you !!